docs/_includes/tk-vred

```diff
@@ -1 +1 @@
-2020.3 — 2022
\ No newline at end of file
+2020.3 — 2023
```

Comment:
Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

Does this apply here?